.eslintrc.js

```diff
@@ -0,0 +1,109 @@
+module.exports = {
+  extends: [
+    'standard',
```

Comment:
I thought one of these rules dealt with quote style, but apparently some files got through the net, so can you add a rule to enforce strings in single-quotes, please?